src/main/java/be/seeseemelk/mockbukkit/inventory/ItemStackMock.java

```diff
@@ -297,4 +309,9 @@ private static void handleLegacyEnchantmentsForDeserialization(@NotNull Map<Stri
 		}
 	}
 
+	@Override
+	public int hashCode() {
+		return Objects.hash(type, amount, itemMeta, durability);
+	}
```

Comment:
We should also override equals method

Reply:
after a bit of talk on the discord  #1058 will cover that part, as it did so way beter than my case, i think i will revert the hashCode change for better merging later on